### ISSUE TITLE
Show type of visitor

### ISSUE THREAD

**Issue by Macro-Jackson** [simpleanalytics/roadmap#194]
I'd really like to see a separation between bots/spiders and visitor's. Right now it will be hard to see if my visitor's are increasing or decreasing as a result of my SEO strategy & actions.

**Comment by adriaanvanrossum** [comment]
We block all bots and spiders by default. So the stats you see are humans (this is different in Google Analytics for example). Does this answer your question?

**Comment by Macro-Jackson** [comment]
Yes and no. Just went live. So not a lot of data to validate on. But right now there are tons non-human page views. For me that's a bot or a spider. I know that because of the URLs viewed are my sitemaps.xml. 
So I guess you mean known spiders and bots are blocked. And my question was I guess a bit more wide than that.

**Comment by adriaanvanrossum** [comment]
That's seems like a bot indeed. Hm, I wonder if we can improve our bot blocking. We have one extra blocking feature on the roadmap: https://github.com/simpleanalytics/roadmap/issues/22.

Do you have a way of figuring out the IP of that bot? We can manually block it to keep your stats (and of others) more accurate.

**Comment by Macro-Jackson** [comment]
Let me watch for a bit longer time. So I can give more clear input. I'm not sure how extensive this problem is. Maybe it's a one time occurrence.

**Comment by adriaanvanrossum** [comment]
Would love to make our bot blocking better. Will keep this issue open for a while.
